pages/a/monitor-your-cratedb-cluster.html

```diff
@@ -0,0 +1,71 @@
+
+### Memory (heap)
+
+Memory is tricky in Java applications since there is runtime garbage collection which is hard to predict sometimes. However, any query (or insert/update/delete) requires enough memory to run, otherwise our [CircuitBreaker](/docs/reference/configuration.html#query-circuit-breaker) causes the query to fail in order to protect the user from OutOfMemory exceptions. The CircuitBreaker's limits can be adjusted but it's usually recommended to optimize the queries and see if there is an underlying problem. In general the amount of memory provided should reflect the query load (also, see 'Queries' above) - concurrency and result sets as well as client connections should be taken into account!
+
+
+### Disk (I/O)
+
+As mentioned above the disk is CrateDB's data store and optimized indices aside, the disk will be utilized for each query and especially [COPY FROM](/docs/reference/sql/reference/copy_from.html) can saturate the disk's IO capacities. Additionally, with growing data sizes and replication, monitoring disk space and I/O performance is critical to ensure that a cluster continues to serve reliably. Particularly the disk usage is something CrateDB can be picky about and for larger disks it's recommended to set the [watermarks](/docs/reference/configuration.html#disk-based-shard-allocation) accordingly.
+
+## What Now?
+
+Monitoring the cluster's CPU, memory, and disk is only the first step, even more important is to act accordingly. Of course the actions are highly dependent on what the business case for this particular CrateDB cluster is and what the goals are. However, in general there are only a few directions available:
+
+- **Mistakes:** Mistakes happen and sometimes the heap size is not set properly or an existing table setup exhibits unexpected behavior from testing. The first step should always be to find out if something changed (recently) and if everything [is set up properly](/docs).
+- **Optimize:** This is the most important point. By choosing the number of shards right and having suitable partitions and routing set up, along with any customized [configuration](/docs/reference/configuration.html) and optimized queries will get the most out of a cluster.
+- **Scale out:** CrateDB is horizontally scalable and adding machines can help since the same number of shards will be worked on by a larger number of nodes.
+- **Ask:** We are happy to help 😊. We will respond on [StackOverflow](https://stackoverflow.com/tags/crate) or if you feel it's a bug open a ticket on [GitHub](https://github.com/crate/crate). For production issues, we recommend our [Enterprise Support plans](/enterprise/support) available!
```

Comment:
We should add something like "Get a feeling for what is right". Administrators who monitor stuff should now their default values in order to recognize when something is wrong.